### PR TITLE
Update graveyard.json - Google Commerce Search

### PR DESCRIPTION
**Pull request by rifkiamil**
Not sure if selected correct branch - https://github.com/codyogden/killedbygoogle/issues/964#issuecomment-876797363

### CODE REVIEW

File: graveyard.json [hunk 1]
```diff
@@ -1,4 +1,12 @@
 [
+  {
+    "dateClose": "2012-07-01",
+    "dateOpen": "2009-11-05",
+    "description": "Hosted enterprise product search to power online retail stores and e-commerce websites with the speed and accuracy customers expect from Google.",
```
Comment (codyogden) [inline graveyard.json:5]:
```suggestion
    "description": "Google Commerce Search was an enterprise search service that powered online retail stores and e-commerce websites that improved speed and accuracy.",
```

**Comment by codyogden** [comment]
Thank you, Rif!

**Comment by rifkiamil** [comment]
@codyogden  my first time doing Pull Requests on an open-source project. Is the config of the build process public, so I can understand when this PR would hit the main branch and show on the public website https://github.com/codyogden/killedbygoogle/tree/main ?

**Comment by codyogden** [comment]
@rifkiamil Ope. Looks like you didn't PR into the `main` branch. Not to worry. I'll merge the upstream branch you chose.

Here's GitHub's [guide to creating a pull request](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request).

**Pull request merged**